style.css

```diff
@@ -65,8 +61,7 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
```

Comment:
@sudheerdarla Are you working on this or should I just merge? Either way is fine with me.